js/webpack.config.js

```diff
@@ -9,7 +9,7 @@ var loaders = [
           'babel-preset-stage-0',
         ].map(require.resolve),
     },
-    exclude: /node_modules/
+    exclude: /node_modules(?!(\/|\\)@jupyter-widgets(\/|\\)jupyterlab-manager)/
```

Comment:
If I import the jupyterlab-manager WidgetManager  class from node-modules it comes as es5 class and cannot be extended using the es6 extends syntax. If instead I add it to our babel transpilation process it is possible to extend it as babel sees it as es6 class.

Reply:
wow, gotcha

Reply:
This puzzled me a bit, but it is the other way around. The output was an ES6 class (https://github.com/jupyter-widgets/ipywidgets/blob/2b91eac459bf4929c3b92794bfb0dc7deef2a353/packages/jupyterlab-manager/src/tsconfig.json#L12) which babel then tried to inherit from, and tried to construct without using new.
I've not configured babel 7 using more modern target browsers such that it output es6 classes for voila as well. The bundle size also seems to have halved.

Reply:
Apologies, I should have suggested a babel upgrade as you did in your commit. I preferred not to as I am not a maintainer of this repo and such a change could have challenge the acceptance of this PR.